Modules/vc-module-pricing/VirtoCommerce.PricingModule.Web/Scripts/blades/assignment-list.js

```diff
@@ -159,7 +161,12 @@ angular.module('virtoCommerce.pricingModule')
 
                         var selectedRows = $scope.gridApi.selection.getSelectedRows();
                         exportDataRequest.dataQuery.objectIds = [];
+                        exportDataRequest.dataQuery.isAllSelected = true;
+                        exportDataRequest.dataQuery.isAnyFilterApplied = false;
+
                         if (selectedRows && selectedRows.length) {
+                            exportDataRequest.dataQuery.isAllSelected = false;
+                            exportDataRequest.dataQuery.isAnyFilterApplied = true;
```

Comment:
probably no need in this line, as in case of selected objectIds it does not matter.